### PR TITLE
updated patterns and partitions

### PR DESCRIPTION
**Pull request by mslacken**
- changed required pattern from minimal_base to base (bsc#1090282) 
- added subvolume description to installation.xml to make btrfs
  available on guided partitioner (bsc#1090380)

### CODE REVIEW

**Review by imobachgs — APPROVED** [review]

File: control/installation.xml [hunk 1]
```diff
@@ -67,11 +67,65 @@
                     <volume>
                       <mount_point>/</mount_point>
                       <fs_type>xfs</fs_type>
-                      <fs_types>xfs,ext3,ext4</fs_types>
+                      <fs_types>xfs,ext3,ext4,btrfs</fs_types>
                       <desired_size config:type="disksize">16 GiB</desired_size>
                       <min_size config:type="disksize">4 GiB</min_size>
                       <max_size config:type="disksize">32 GiB</max_size>
                       <weight config:type="integer">20</weight>
+			<snapshots config:type="boolean">true</snapshots>
```
Comment (imobachgs) [inline control/installation.xml]: Check indentantion, please.

**Review by imobachgs — CHANGES_REQUESTED** [review]

File: control/installation.xml (hunk 1 above)
Comment (mslacken) [reply control/installation.xml]: are there rules for that?

**Comment by mslacken** [comment]
updated intendation and comment

**Comment by mslacken** [comment]
- updated intendation

File: control/installation.xml [hunk 2]
```diff
@@ -89,7 +141,7 @@
                       <max_size config:type="disksize">unlimited</max_size>
                       <weight config:type="integer">70</weight>
                       <!-- If this volume is disabled, we want "/" to become greedy (unlimited
-                           max), since it will contain /var/lib/libvirt -->
+                           max), because big simultion will go here -->
```
Comment (imobachgs) [inline control/installation.xml]: I guess *simultion* is a typo, right?

**Review by imobachgs — CHANGES_REQUESTED** [review]
It looks much better know, although I've spotted a typo in a comment. Could you check it, please?

**Review by imobachgs — APPROVED** [review]

**Pull request merged**